apps/site/package.json

```diff
@@ -3,9 +3,10 @@
   "type": "module",
   "scripts": {
     "scripts:release-post": "cross-env NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "dev": "cross-env NODE_NO_WARNINGS=1 next dev",
-    "serve": "npm run dev",
-    "build": "cross-env NODE_NO_WARNINGS=1 next build --turbopack",
+    "dev": "npm run build-blog-data:watch & sleep 1 && cross-env NODE_NO_WARNINGS=1 next dev",
+    "serve": "npm run build-blog-data:watch & sleep 1 && npm run dev",
```

Comment:
Isn't this running the build twice? Once for serve, and again when serve calls dev?

Reply:
fixed, thanks for noticing it 🙂